### PR TITLE
[MM-60812] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by Camillarhi**
#### Summary

Replace FormattedMarkdownMessage in `webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx` with FormattedMessage
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28468
Jira https://mattermost.atlassian.net/browse/MM-60812

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @Camillarhi!

File: webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx [hunk 1]
```diff
@@ -148,17 +147,23 @@ export function TeamProfile({team, isArchived, onToggleArchive, isDisabled, save
...
                         <div className='team-desc-col'>
                             <div className='row row-bottom-padding'>
-                                <FormattedMarkdownMessage
-                                    id='admin.team_settings.team_detail.teamName'
-                                    defaultMessage='**Team Name**:'
+                                <FormattedMessage
+                                    id='admin.teamSettings.teamDetail.teamName'
+                                    defaultMessage='<b>Team Name</b>:'
+                                    values={{
+                                        b: (chunks: string) => <b>{chunks}</b>,
```
Comment (larkox) [inline webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx:154]: @M-ZubairAhmed In the example you used `<strong>` and here we are using `<b>`. Should we follow any particular standard on this? or is it a case by case basis?

**Review by larkox — APPROVED** [review]
Looks good to me. Just one think to verify with @M-ZubairAhmed

**Comment by M-ZubairAhmed** [comment]
@Camillarhi can check the conflicts?

File: webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx (hunk 1 above)
Comment (M-ZubairAhmed) [reply webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx:154]: Oh lets open a thread in dev channel for this

**Comment by Camillarhi** [comment]
> @Camillarhi can check the conflicts?

The conflicts has been resolved

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**